### DISCUSSION TITLE
App Name Ideas

### DISCUSSION THREAD

**Discussion by isaaclee12** [MatthewBozin/Virtual-Office#13] [Ideas]
This is a list of possible names for the app. I am aggregating them all here for recording purposes. Might archive this later when a name is decided on.

- Current name ideas:
- Tech Collab
- Dev Collab
- 100Devs
- 1000Devs
- 10000Devs
- 100000Devs
- Twogether 2: Togetherer 4ever
- reNO (dvkr's idea)
- Remon't
- Zoomeeteams
- JavaScript Sucks
- Leon's Virtual Office Building, in which the rent is still too damn high regardless 
- Gather
- Devs Space
- Office Gather
- Tech Gather
- Devs Workspace
- Gathr.io
- Gaderin.com (Cornelius' domain name)
- Gabble
- LocalHost2121
- GatherSpace
- StudioGather
- GatherWise
- RoomCollab
- CallLab
- ChatLab
- ChattyLab
- 2Gathr

**Comment by Caleb-Cohen** [comment]
Gathr.io is my favorite so far

**Reply by Natashka333** [reply]
- LocalHost2121 
- GatherSpace
- StudioGather
- GatherWise
- RoomCollab
- CallLab
- ChattyLab  

![Screen Shot 2023-01-20 at 6 36 30 PM](https://user-images.githubusercontent.com/102708144/213829486-b9853367-18e9-48ee-a81d-fb57a665a76c.png)

**Reply by Caleb-Cohen** [reply]
> - LocalHost2121 
> 
> - GatherSpace
> 
> - StudioGather
> 
> - GatherWise
> 
> - RoomCollab
> 
> - CallLab
> 
> - ChattyLab  
> 
> 
> 
> ![Screen Shot 2023-01-20 at 6 36 30 PM](https://user-images.githubusercontent.com/102708144/213829486-b9853367-18e9-48ee-a81d-fb57a665a76c.png)
> 
> 
> 
> 
> 
> 

Chatty lab 😂

**Comment by MatthewBozin** [comment]
I like Gathr.io

Also, for consideration: 2Gathr?

**Reply by Caleb-Cohen** [reply]
Gathr.io and ChatLab/chattyLab is S-tier
